### ISSUE TITLE
Include overhead from freelist in Database::stats

### ISSUE THREAD

**Issue by casey** [cberner/redb#197]
Does DbStats include information about how much space is used by the tree holding free pages? If not, then it should include it, either in `metadata_bytes`, or in a separate field.

**Comment by cberner** [comment]
Yes, but most of it is in the `stored_leaf_bytes` field. I'll fix that. It ends up there because internally the freelist is stored just like a normal table
